apps/web/src/components/layout/components/HeaderNav.tsx

```diff
@@ -96,7 +106,7 @@ export function HeaderNav({}: Props) {
         },
       },
     ]);
-  }, [colorScheme]);
+  }, [colorScheme, Icon, addItem, logout, themeTitle, toggleColorScheme]);
```

Comment:
Missing dependencies for the hook. Warning risen once rules were enabled properly.